clang/include/clang/StaticAnalyzer/Checkers/Checkers.td

```diff
@@ -1730,6 +1721,21 @@ def UnixAPIPortabilityChecker : Checker<"UnixAPI">,
 
 } // end optin.portability
 
+
+//===----------------------------------------------------------------------===//
+// Taint checkers.
+//===----------------------------------------------------------------------===//
+
+let ParentPackage = TaintOptIn in {
+
+def TaintMallocChecker: Checker<"TaintMalloc">,
+  HelpText<"Check for memory allocations, where the size parameter "
+           "might be a tainted (attacker controlled) value.">,
+  Dependencies<[DynamicMemoryModeling]>,
```

Comment:
Once the `TaintPropagation` checker will be brought out of alpha, it should be added as a dependency of this checker.

Reply:
Why can't we make it a dependency now? (I don't have a strong opinion, I'm more just curious)

Reply:
I added the GenerictaintChecker as a dependency too. I think it makes sense to add it now so that we dont forget it later.